### ISSUE TITLE
Wrong weather

### ISSUE THREAD

**Issue by aradzil** [GuyKh/ims-custom-component#27]
Hi Tomer,
Thank you very much for this.
There are some problems with the weather as I can see for now.
My weather is listed as sunny with 20 degrees but at the same time, it's raining outside.
Please see the following picture:
![image](https://user-images.githubusercontent.com/39214425/225254315-764f62e0-fd59-4e3d-89b5-2b1bb90d7c30.png)
In addition, the rain sensor indicates "Not Raining" (possibly because of the first issue).
My city is Modiin (code 19).
Thanks,
Arad.

**Comment by aradzil** [comment]
![image](https://user-images.githubusercontent.com/39214425/225255180-fd731fa4-da30-47ac-99c6-430f187022a9.png)

**Comment by GuyKh** [comment]
In the first picutre - youre showing a **forecast** (atleast from a day before).
Calling the IMS API, get me this:

https://ims.gov.il/en/now_analysis
```
"19": {
"id": "1781680",
"lid": "19",
"forecast_time": "2023-03-15 12:00:00",
"type": "analysis",
"main_hour": "0",
"heat_stress": "13",
"relative_humidity": "78",
"due_point_Temp": "10",
"rain": null,
"temperature": "14",
"wind_direction_id": "12",
"wind_speed": "22",
"wind_chill": "12",
"weather_code": null,
"heat_stress_level": "0",
"feels_like": "14",
"min_temp": null,
"max_temp": null,
"modified": "2023-03-15 11:56:00",
"created": "2023-03-12 07:51:15",
"u_v_index": "2",
"u_v_level": "L",
"u_v_i_max": null,
"u_v_i_factor": null
},
```

As you can see - `"rain": null` -- so we're limited to how well the IMS updates its data.
The sensor data is as good as the IMS data.
Don't expect it to be a 100% accurate sensor

**Comment by aradzil** [comment]
Hi Guy,
Thanks.
Regarding the first picture, I highlighted that it's from today's date, you can see it matches. Maybe I don't look at the right place?

**Comment by GuyKh** [comment]
I think you're using an outdated version of the component (i.e. not 0.1.4).
I fixed in 0.1.4 the fact that it shouldn't present outdated forecasts.

Moreover, as you can see in the picture the forecast is for 2023-03-15 **22:00** - so that's at night; Although "Sunny" is a bad naming for this, but since there is no "Clear" - only "Clear Night" in HomeAssistant, while the "Clear" sign is for both day and night -- I selected to use "Sunny" (even at night) -- and right now it's clear (atleast here in Tel Aviv)

**Comment by aradzil** [comment]
Thanks!
I'm closing this issue.
